tests/test_recipes.py

```diff
@@ -41,6 +41,14 @@ def test_import_seq_from_recipe():
         pytest.fail("{} raised".format(ImportError.__name__))
 
 
+def test_import_recipes():
+    """Test imports works both for full import paths and for
+    `app_name.recipe_name` strings."""
+    assert baker.prepare_recipe("generic.Dog"), baker.prepare_recipe(
+        "tests.generic.Dog"
+    )
```

Comment:
They should be lower case because they're recipes names

```suggestion
    assert baker.prepare_recipe("generic.dog"), baker.prepare_recipe(
        "tests.generic.dog"
    )
```